query-engine/dmmf/src/tests/test-schemas/odoo.prisma

```diff
@@ -0,0 +1,3567 @@
+datasource db {
```

Comment:
I'd prefer not committing large schemas twice. Would it be ok to `include_str!("../../schema-builder/benches/odoo.prisma")`? Or we can put schemas in a shared, different place.